src/write.rs

```diff
@@ -476,7 +483,177 @@ where
                 Absorb::absorb_second(w_inner, op, &*r_handle);
             }
         } else {
-            self.oplog.extend(ops);
+            // Only try to compress if it is enabled, else use efficient fallback.
+            if T::MAX_COMPRESS_RANGE > 0 {
```

Comment:
I'd like to invert the logic here.
```rust
if T::MAX_COMPRESS_RANGE == 0 {
    // .. easy case ..
    return;
}
// .. hard case ..
```

Reply:
Done: 000d51264f3ac991359f656a0173eba70b3a3048